library/rsa.c

```diff
@@ -500,9 +500,24 @@ int mbedtls_rsa_set_padding(mbedtls_rsa_context *ctx, int padding,
 }
 
 /*
- * Get length in bytes of RSA modulus
+ * Get padding mode of initialized RSA context
  */
+int mbedtls_rsa_get_padding_mode(const mbedtls_rsa_context *ctx)
+{
+    return ctx->padding;
+}
 
+/*
+ * Get hash identifier of mbedtls_md_type_t type
+ */
+int mbedtls_rsa_get_hash_id(const mbedtls_rsa_context *ctx)
```

Comment:
We don't use `hash_id` elsewhere in the API (only as a variable name which is not part of the API). Looking for a precedent, there are X.509 functions called `..._set_md_alg`, so I think a good name would be `mbedtls_rsa_get_md_alg`.